lib/matplotlib/backends/backend_qt5.py

```diff
@@ -1034,9 +1034,9 @@ def trigger_manager_draw(manager):
     def mainloop():
         old_signal = signal.getsignal(signal.SIGINT)
         # allow SIGINT exceptions to close the plot window.
-        signal.signal(signal.SIGINT, signal.SIG_DFL)
+        if old_signal: signal.signal(signal.SIGINT, signal.SIG_DFL)
```

Comment:
the style checker is not going to like this (you need to split this over two lines), but other than that, the fix looks reasonable.

Reply:
thanks, fixed.